.prettierignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+package-lock.json
+src/utils/browser-utils.js
```

Comment:
@rootux my only problem is here: browserUtil.js . getUrlSearchParams, the last return
originally its:
return (q[key] = value, q)
but for some reason prettier is changing it to be
return (q[key] = value), q;

any idea?